src/OpenTelemetry.Instrumentation.Process/ProcessMetrics.cs

```diff
@@ -20,42 +20,67 @@
 
 namespace OpenTelemetry.Instrumentation.Process;
 
-internal class ProcessMetrics
+internal sealed class ProcessMetrics
 {
     internal static readonly AssemblyName AssemblyName = typeof(ProcessMetrics).Assembly.GetName();
-    internal static readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
-    private static readonly Diagnostics.Process CurrentProcess = Diagnostics.Process.GetCurrentProcess();
+    internal readonly Meter MeterInstance = new(AssemblyName.Name, AssemblyName.Version.ToString());
```

Comment:
I am wondering, if creating 2 instances of `ProcessMetrics` class, do we expect every instance create its own meter with the same name?

Reply:
My understanding is that in the SDK, [MetricStreamIdentity](https://github.com/open-telemetry/opentelemetry-dotnet/blob/879d3c91a0032eee75061871851e2d4febd6899f/src/OpenTelemetry/Metrics/MetricStreamIdentity.cs) is maintained, so there will be only **one** `MetricStreamIdentity` in this case.

What would happen is both of the callbacks from the instruments would be firing, but there will only be one export.
The second one will hit this https://github.com/open-telemetry/opentelemetry-dotnet/blob/7f71283c0bf2ee0e2c5e3c9617295ddaf20c7a0c/src/OpenTelemetry/Metrics/MetricReaderExt.cs#L51

Please refer to the [testcases](https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/6bfac03ec17a111d21342d6954feab1f696b9b59/test/OpenTelemetry.Instrumentation.Process.Tests/ProcessMetricsTests.cs) for the experiments I've done.
Thank you.

Reply:
> there will be only one MetricStreamIdentity in this case.

I doubt that. `MetricStreamIdentity` is maintained at a `MetricReader` level within the SDK. In the scenario, where we have two instances of `ProcessMetrics` class due to multiple MeterProviders we would also have multiple readers each maintaining their own set of `MetricStreamIdentity`.

Reply:
Hmm... I think each reader has its own storage with the current implementation - usage of instance for `ProcessMetrics`; 
so it may not be required to maintain MetricStreamIdentity at the reader level.